src/main/java/org/tron/core/net/node/NodeImpl.java

```diff
@@ -605,7 +609,8 @@ public synchronized void disconnectInactive() {
     getActivePeer().forEach(peer -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Specify a type for: 'peer' ([squid:S2211](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2211))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-0olHK86Hco_1s2&open=AWalU-0olHK86Hco_1s2)

[ISSUE_KEY]: AWalU-0olHK86Hco_1s2